pkg/monitortests/testframework/watchevents/event.go

```diff
@@ -150,6 +150,10 @@ func recordAddOrUpdateEvent(
 				}
 			}
 		}
+	case "CABundleUpdateRequired", "SignerUpdateRequired", "TargetUpdateRequired", "CertificateUpdated", "CertificateRemoved":
+		message = message.WithAnnotation(monitorapi.AnnotationInteresting, "true")
+	case "CertificateUpdateFailed":
+		message = message.WithAnnotation(monitorapi.AnnotationPathological, "true")
```

Comment:
AnnotationPathological is for events that re-occur over the threshold of 20x in a job run, it's not intended as a mechanism for severity. In your case it almost feels like you may want to add an invariant test that looks for these CertificateUpdateFailed events and fails, or perhaps just flakes, a test.

Reply:
so, instead of displaying these events, we should have a synthetic test which ensures these events didn't happen? Yeah, that would work I guess

Reply:
You can absolutely display them, just mark them interesting, not pathological.

Based on your changeover to using patho I assumed you consider this event quite serious, and if you did write a test that flakes if it sees one of those, it would help you see fleet wide how often that is happening.

Reply:
Okay, added it to interesting list. Will create a flaking test to find these events as a followup